CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: BadInput traces shown when alarm is already raised ("flapping" effect) (#3860)
```

Comment:
There is a conflict in this file.

Maybe CHANGES_NEXT_RELEASE gets update more times before this PR can be merged (so the conflict can happen again) but I think it would be a good idea to solve it now (which, in addition, will case a beneficial sync with master branch).